usr/lib/zypp/plugins/urlresolver/susecloud

```diff
@@ -31,6 +31,14 @@ utils.set_proxy()
 class SUSECloudPlugin(Plugin):
     def RESOLVEURL(self, headers, body):
         """Convert the URL for the repo"""
+        if not utils.is_plugin_allowed():
+            logging.warning('URL resolver plugin not available.')
+            self.error(
+                {'CREDENTIAL_ERROR': 'NOTAVAILABLE'},
+                'URL resolver plugin not available'
+            )
+            return
```

Comment:
We want to get out of the way, not sabotage the update process. zypper expects an answer this just breaks zypper up